tests/foreman/virtwho/test_api_virtwho.py

```diff
@@ -151,22 +157,26 @@ def test_positive_deploy_configure_by_script(self):
 
         :CaseImportance: High
         """
-        name = gen_string('alpha')
-        args = self._make_virtwho_configure()
-        args.update({'name': name})
-        vhd = entities.VirtWhoConfig(**args).create()
-        self.assertEqual(vhd.status, 'unknown')
-        script = vhd.deploy_script()
+        assert virtwho_config.status == 'unknown'
+        script = virtwho_config.deploy_script()
         hypervisor_name, guest_name = deploy_configure_by_script(
             script['virt_who_config_script'], debug=True
         )
-        self.assertEqual(
-            entities.VirtWhoConfig().search(query={'search': 'name={}'.format(name)})[0].status,
-            'ok',
+        assert (
+            entities.VirtWhoConfig()
+            .search(query={'search': 'name={}'.format(virtwho_config.name)})[0]
+            .status
+            == 'ok'
```

Comment:
I'm not a fan of this style... especially hanging comparison. personally I would use something like:
```suggestion
virt_who_instance = entities.VirtWhoConfig().search(
    query={'search': 'name={}'.format(virtwho_config.name)}
)[0]
        assert virt_who_instance.status == 'ok'
```
Also, you can make it little shorter with f-string.

Reply:
Changed the style, changed to f-string